scripts/functions/installer

```diff
@@ -1019,150 +1021,175 @@ setup_user_profile_check()
 setup_user_profile_detect()
 {
   etc_profile_file="/etc/profile.d/rvm.sh"
-  search_list_mksh=( "$HOME/.mkshrc" "$HOME/.profile" )
-  search_list_bash=( "$HOME/.bashrc" "$HOME/.bash_profile" "$HOME/.bash_login" )
-  search_list_zsh=( "${ZDOTDIR:-${HOME}}/.zshenv" "${ZDOTDIR:-${HOME}}/.zprofile" "${ZDOTDIR:-${HOME}}/.zshrc" "${ZDOTDIR:-${HOME}}/.zlogin" )
```

Comment:
The different list was used for searching and different for adding. But then they were merged anyway. So I simplified this and use only one list now.